php-frontend/src/test/java/org/sonar/php/cfg/ControlFlowGraphTest.java

```diff
@@ -360,13 +360,13 @@ public void test_branching_tree() {
   @Test
   public void simple_for() {
     verifyBlockCfg("" +
-      "before(succ = [forCond], elem = 1);" +
-      "for ($i=1 ; forCond( succ = [forBody, END], elem = 1); $i++) {" +
-      "  forBody( succ = [forCond], elem = 1 );" +
+      "before(succ = [cond], elem = 1);" +
```

Comment:
why 1 elem?

Reply:
because it has one function call: `before`